src/main/modules/properties-volume/index.ts

```diff
@@ -7,12 +7,16 @@ export class PropertiesVolume {
   enableFor(server: Application): void {
     require('dotenv').config();
     set(config, 'session.redis.key', process.env.REDIS_ACCESS_KEY);
+    set(config, 'ams.flowKey', process.env.AMS_FLOW_KEY);
+    set(config, 'ams.flowUrl', process.env.AMS_FLOW_URL);
     if (server.locals.ENV !== 'development') {
       propertiesVolume.addTo(config);
       this.setSecret('secrets.pre-hmctskv.AppInsightsInstrumentationKey', 'appInsights.instrumentationKey');
       this.setSecret('secrets.pre-hmctskv.redis6-access-key', 'session.redis.key');
       this.setSecret('secrets.pre-hmctskv.apim-sub-portal-primary-key', 'pre.apiKey.primary');
       this.setSecret('secrets.pre-hmctskv.apim-sub-portal-secondary-key', 'pre.apiKey.secondary');
+      this.setSecret('secrets.pre-hmctskv.pp-authorization', 'pre.ams.flowKey');
+      this.setSecret('secrets.pre-hmctskv.ams-flow-url', 'pre.ams.flowUrl');
```

Comment:
```suggestion
      this.setSecret('secrets.pre-hmctskv.pp-authorization', 'ams.flowKey');
      this.setSecret('secrets.pre-hmctskv.ams-flow-url', 'ams.flowUrl');
```

Reply:
I don't think I need this, if you see `config/default.json` I have `ams` as its own section

Reply:
but it's not inside of the `pre` block. this will set `pre.ams.flowKey` but you want it to set `ams.flowKey` as u do earlier in the file:
`    set(config, 'ams.flowKey', process.env.AMS_FLOW_KEY);`